using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -277,7 +331,7 @@ If your application uses more complicated URLs, such as Angular URLs, we recomme
 {% tab title="iOS" %}
 ### appendToURL <a id="appendToUrl-ios"></a>
 
-Appends Adobe visitor data to the query component of the given URL
+This `appendToURL()` API appends Adobe visitor information to the query component of the given URL
```

Comment:
Reword to say:
"This  API appends Adobe visitor information to the query component of the given URL.